docs/openapi-ts/configuration.md

```diff
@@ -105,28 +105,68 @@ You might not need a `node` client. Fetch API is [experimental](https://nodejs.o
 
 By default, `openapi-ts` will automatically format your client according to your project configuration. To disable automatic formatting, set `format` to false
 
-```js{2}
+::: code-group
+
+```js{2} [prettier (default)]
+export default {
+  format: 'prettier',
+  input: 'path/to/openapi.json',
+  output: 'src/client',
+}
+```
+
+```js{2} [biome]
+export default {
+  format: 'biome',
+  input: 'path/to/openapi.json',
+  output: 'src/client',
+}
+```
+
+```js{2} [off]
 export default {
   format: false,
   input: 'path/to/openapi.json',
   output: 'src/client',
 }
 ```
 
+:::
+
 You can also prevent your client from being processed by formatters by adding your output path to the tool's ignore file (e.g. `.prettierignore`).
 
 ## Linting
 
 For performance reasons, `openapi-ts` does not automatically lint your client. To enable this feature, set `lint` to true
 
-```js{3}
+::: code-group
+
+```js{3} [eslint]
+export default {
+  input: 'path/to/openapi.json',
+  lint: 'eslint',
+  output: 'src/client',
+}
+```
+
+```js{3} [biome]
 export default {
   input: 'path/to/openapi.json',
-  lint: true,
+  lint: 'biome',
   output: 'src/client',
 }
 ```
 
+```js{3} [off (default)]
```

Comment:
Can you put the default value first in the group? Not really a fan of putting disabled in the heading, but not feeling too strongly about it

```suggestion
```js{3} [disabled]
```